client/internal/peer/conn.go

```diff
@@ -234,6 +237,17 @@ func (conn *Conn) reCreateAgent() error {
 		return err
 	}
 
+	err = conn.agent.OnSuccessfulSelectedPairBindingResponse(func(p *ice.CandidatePair) {
+		err := conn.statusRecorder.UpdateLatency(conn.config.Key, p.Latency())
+		if err != nil {
+			log.Debugf("failed to update latency for peer %s: %s", conn.config.Key, err)
+			return
+		}
+	})
+	if err != nil {
+		return err
```

Comment:
Can you annotate this error with some context? Makes debugging from the logs easier

Reply:
```suggestion
		return fmt.Errorf("select pair binding response: %w", err)
```